app/src/main/java/com/bignerdranch/android/androidprojecttms/Order.kt

```diff
@@ -0,0 +1,15 @@
+package com.bignerdranch.android.androidprojecttms
+
+data class Order(
+    private val idOrder: Int,
+    private val consumer: String,
+    private val product: String,
+    private val address: String,
+    private val price: Double,
+) {
+    fun getIdOrder(): Int = idOrder
```

Comment:
`val` without any modifier gives you the same result: only public getter available. 
Reasonable for all fields.